recipe/conda_build_config.yaml

```diff
@@ -214,7 +214,7 @@ bzip2:
 cairo:
   - 1.14
 curl:
-  - 7.44
+  - 7.48
```

Comment:
No, this should either be `7.59` or `7` because osx packages built with `7.59` are not compatible with `7.48`

Reply:
I see, I was thinking about this wrong. On Linux it needs to be at least 7.56 if we already have things building against that, which we probably do.

What's different on OSX?

Reply:
So it doesn't seem like curl bumps the dylib version every time: everything from 7.48 through 7.58 gives me
```
@rpath/libcurl.4.dylib (compatibility version 9.0.0, current version 9.0.0)
```
which is kind of strange, because 7.56 did add symbols. But since 7.59 has version 10.0.0, looks like we need to use that.